src/lightning/pytorch/loggers/csv_logs.py

```diff
@@ -95,7 +96,7 @@ class CSVLogger(Logger, FabricCSVLogger):
     def __init__(
         self,
         save_dir: _PATH,
-        name: str = "lightning_logs",
+        name: Optional[str] = "lightning_logs",
```

Comment:
The same change needs to be applied to `src/lightning/fabric/loggers/csv_logs.py` as well

Reply:
Sorry, I missed this. Done in 45280259ccb1b8087c749e444140e01613d910f6